src/cli/cli_br.cpp

```diff
@@ -381,11 +379,28 @@ template <> otError Br::Process<Cmd("nat64prefix")>(Arg aArgs[])
  * @code
  * br prefixtable
  * prefix:fd00:1234:5678:0::/64, on-link:no, ms-since-rx:29526, lifetime:1800, route-prf:med,
- * router:ff02:0:0:0:0:0:0:1
+ * router:ff02:0:0:0:0:0:0:1 (M:0 O:0 Stub:1)
  * prefix:1200:abba:baba:0::/64, on-link:yes, ms-since-rx:29527, lifetime:1800, preferred:1800,
- * router:ff02:0:0:0:0:0:0:1
+ * router:ff02:0:0:0:0:0:0:1 (M:0 O:0 Stub:1)
  * Done
  * @endcode
+ * @par
+ * Get the discovered prefixes by Border Routing Manager on the infrastructure link.
+ *
+ * Info per prefix entry:
+ * - The prefix
+ * - Whether or not the prefix is on-link (on-link or route)
+ * - Milliseconds since last received Router Advertisement containing this prefix
+ * - Prefix lifetime in seconds
+ * - Preferred lifetime in seconds only if prefix is on-link
+ * - Route preference (low, med, high) only if prefix is route (not on-link)
+ * - The router IPv6 address which advertising this prefix
+ * - Flags in received Router Advertisement header:
+ *   - M: Managed Address Config flag
+ *   - O: Other Config flag
+ *   - Stub: Stub Router flag (indicates whether the router is a stub router)
+
+
  * @par api_copy
  * #otBorderRoutingGetNextPrefixTableEntry
```

Comment:
The description for this API doesn't exactly align with the command... can you switch this to a @sa only

`@sa otBorderRoutingGetNextPrefixTableEntry`

Reply:
Change in new push.